data/hai/hai outfits.txt

```diff
@@ -440,6 +440,7 @@ outfit "Quantum Keystone"
 	"outfit space" -1
 	"quantum keystone" 1
 	description "This chunk of rock from the mountains on the Hai homeworld is apparently nothing more than a good luck charm, but the Hai have decorated their ships with them since the height of their empire. More superstitious Hai claim that the stones somehow reduce the nausea induced by hyperspace jumps by making travel between star systems easier, and many wealthy Hai refuse to book passage on a ship without one."
+	description "	This keystone comes with a fancy certificate of authenticity certifying its origins on Hai Home, including a decorative frame. The packaging suggests mounting the framed certificate in a prominent location near the main airlock or public common area."
```

Comment:
Possible longer suggestion:
```suggestion
	description "	This keystone comes with a package including; a holo-film certificate of authenticity certifying its origins on Hai-home, a decorative but robust frame, and a tiny plaque with the keystone logo and sale number stamped on it. The packaging suggests mounting the framed certificate in a prominent location near the main airlock or public common area."
```
Minimum viable product:
```suggestion
	description "	This keystone comes with a fancy certificate of authenticity certifying its origins on Hai-home, and a robust, decorative frame for displaying it. The packaging suggests mounting the framed certificate in a prominent location near the main airlock or public common area."
```

Reply:
Both of those look good; but I *am* curious as to the importance of the "robust" descriptor.

Reply:
Longer one I'm thinking that should probably be a ':' instead of ';' , no?

I do prefer that one though.